.github/workflows/ci.yml

```diff
@@ -1,23 +1,41 @@
+name: CI
 on: [push, pull_request]
 
 jobs:
-  build:
-    runs-on: ubuntu-22.04
+  lint:
     strategy:
       fail-fast: false
       matrix:
-        go: [ '1.17', '1.18', '1.19' ]
-        os: [ ubuntu-22.04, windows-2022 ]
-    name: ${{ matrix.os }} / Go ${{ matrix.go }}
+        os: [ 'ubuntu-22.04' ]
+
+    name: ${{ matrix.os }} / Lint
+    runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v3
+      - uses: actions/setup-go@v4
+        with:
+          go-version: '1.20'
```

Comment:
Any concern letting it follow go.mod by `go-version-file: go.mod` instead of hardcoding to 1.20?

Reply:
SGTM, as long as it is pinned somewhere. :)